packages/es-components/src/components/controls/dropdown/DropdownV2.md

```diff
@@ -0,0 +1,40 @@
+A `Dropdown` component works exactly like a native `<select>` tag with style attached. It supports any HTML attributes that a native `<select>` tag supports.
+
+```
+const Control = require('../Control').default;
+
+<Control>
+  <Label for="dropdown-1">Select an operating system</Label>
+  <DropdownV2
+    id="dropdown-1"
+    onChange={({ target }) => console.log(target.value)}
+  >
+    <option value="">--</option>
+    <option value="win">Windows</option>
+    <option value="mac">Mac</option>
+    <option value="linux">Linux</option>
+  </DropdownV2>
+</Control>
+```
+
+Wrap the dropdown in a `Control` with `inline` passed for the `orientation` prop to render the label and input horizontally.
+
+```
+const Control = require('../Control').default;
+const AdditionalHelp = require('../AdditionalHelp').default;
+
+<Control orientation="inline" validationState="danger">
+  <Label for="dropdown-2">Choose your favorite vehicle</Label>
+  <DropdownV2 id="dropdown-2">
+    <option value="">--</option>
+    <option value="car">Car</option>
+    <option value="truck">Truck</option>
+    <option value="suv">SUV</option>
+    <option disabled value="atv">ATV</option>
+    <option value="boat">Boat</option>
+  </DropdownV2>
+  <AdditionalHelp>
```

Comment:
For this example I'd like to see an `id` on AdditionalHelp with an associated `aria-describedby` on the dropdown.